### PR TITLE
Add IRC Ghost Serial Rx Protocol

### PR DESCRIPTION
**Pull request by tonycake**
Enable 'IRC Ghost' protocol for Api version 1.44 and later. Will be accompanied by a similar PR in betaflight to install the driver.

### CODE REVIEW

**Comment by McGiverGim** [comment]
Usually is a good idea to reference the firmware PR: https://github.com/betaflight/betaflight/pull/10217

**Review by McGiverGim — APPROVED** [review]

File: src/js/fc.js [hunk 1]
```diff
@@ -544,6 +544,10 @@ const FC = {
                     result.push('SPEKTRUM SRXL2');
                 }
 
+                if (semver.gte(apiVersion, "1.44.0")) {
```
Comment (Asizon) [inline src/js/fc.js:547]: We are using now a new API_VERSION constant here.

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/js/fc.js (hunk 1 above)
Comment (McGiverGim) [reply src/js/fc.js:547]: True, I forgot!

**Review by McGiverGim — CHANGES_REQUESTED** [review]
Please use the constant as indicated by asizon

File: src/js/fc.js (hunk 1 above)
Comment (tonycake) [reply src/js/fc.js:547]: Thanks guys, just to confirm, you are suggesting replacing the "1.44.0" with API_VERSION_1_44 right?

Comment (McGiverGim) [reply src/js/fc.js:547]: Yes, that is!

**Review by McGiverGim — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Pull request merged**